pubtools/exodus/_tasks/push.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import subprocess
+
+from pubtools.exodus.task import ExodusTask
+
+LOG = logging.getLogger("pubtools-exodus")
+LOG_FORMAT = "%(asctime)s [%(levelname)-8s] %(message)s"
+
+
+class ExodusPushTask(ExodusTask):
+    """Push a directory to the Exodus CDN"""
+
+    def add_args(self):
+        super(ExodusPushTask, self).add_args()
+
+        self.parser.add_argument(
+            "--dry-run",
```

Comment:
Minor - you might want to consider making it so that, instead of explicitly coding in support for \-\-dry-run, instead just pass any remaining arguments onto exodus-rsync.  It can be done by using parse_known_args instead of parse_args (and probably some other ways).

It gives the advantage that you don't need to explicitly write code to handle \-\-dry-run, and also that any additional arguments supported by exodus-rsync will automatically be supported by this command without any code changes.  I can imagine this might save us in some emergency scenario, we'll be kicking ourselves if a failure occurs at a critical time which *could* be worked around just by supplying an additional argument, yet we didn't give ourselves any way to do that.

On the flip side though, that means the caller could start to depend on exodus-rsync implementation details when invoking this command, which isn't ideal.  So it's a trade-off either way.